src/externalServices/storeTrigger/requestCall.ts

```diff
@@ -30,4 +31,21 @@ export class StoreTriggerCall {
     });
     return response.data;
   }
+
+  public async deleteModel(request: DeleteRequest): Promise<StoreTriggerResponse> {
+    this.logger.debug({
+      msg: 'got a request for a new flow',
```

Comment:
Add also that the request is for delete.
```suggestion
      msg: 'sending the delete request to store-trigger',
```

Set this suggestion also in createModel

Reply:
ok